example/composition.yaml

```
@@ -1,17 +1,60 @@
 apiVersion: apiextensions.crossplane.io/v1
 kind: Composition
 metadata:
-  name: function-template-go
+  name: function-environment-configs
 spec:
   compositeTypeRef:
     apiVersion: example.crossplane.io/v1
     kind: XR
   mode: Pipeline
   pipeline:
-  - step: run-the-template
+  - step: environmentConfigs
     functionRef:
-      name: function-template-go
+      name: function-extra-resources
     input:
-      apiVersion: template.fn.crossplane.io/v1beta1
+      apiVersion: extraresources.fn.crossplane.io/v1beta1
```

For consistency with the apiVersion declared below:
```suggestion
      apiVersion: extra-resources.fn.crossplane.io/v1beta1
```